ivi/rohdeschwarz/rohdeschwarzBaseScope.py

```diff
@@ -972,10 +982,49 @@ def _measurement_fetch_waveform(self, index):
         y = [float(yi) for yi in y]
         return list(zip(x, y))
 
+    def _measurement_fetch_waveform_measurement(self, index, measurement_function, ref_channel = None):
+        index = ivi.get_index(self._channel_name, index)
+        meas_source1 = None
+        meas_source2 = None
+        channel_state = self._get_channel_enabled(index)
+        if index < self._analog_channel_count:
+            if measurement_function not in MeasurementFunctionMapping:
+                raise ivi.ValueNotSupportedException()
+            func = MeasurementFunctionMapping[measurement_function]
+            meas_source1 = "CH%d" %(index+1)
```

Comment:
Use the whole scpi wor for channel instead of "CH" as in the rest of the driver. This enhances readability.

Reply:
this is special keyword of scope... used in other parts of code too..